lib/foreman_ansible/register.rb

```diff
@@ -76,6 +76,14 @@
                :resource_type => 'Hostgroup'
     permission :generate_ansible_inventory,
                { :'api/v2/ansible_inventories' => [:schedule] }
+    permission :import_ansible_playbooks,
+               { :ansible_playbooks => [:sync],
```

Comment:
There is only API controller for now, so this line is not needed. Same for the permission below.